src/Table/HeaderCell.less

```diff
@@ -0,0 +1,43 @@
+@import "../colors.less";
+
+
+.Table--header--cell {
+  background-color: @neutral_off_white;
+  border-bottom: 3px solid @neutral_silver;
+  color: @neutral_black;
+  font-size: 16px;
```

Comment:
ah cool, didn't know we were using `rem` units!
should the header be a little larger than the body size, then? 1.25rem?